app/controllers/api/service_dialogs_controller.rb

```diff
@@ -34,6 +34,14 @@ def edit_resource(type, id, data)
       service_dialog
     end
 
+    def copy_resource(type, id, data)
+      service_dialog = resource_search(id, type, Dialog)
+      data['label'] = "Copy of #{service_dialog.label}" unless data.key?('label')
```

Comment:
It might be better not to mutate the data that's passed into this method - perhaps you could `data.dup` or otherwise build the hash that's sent to `deep_copy`?